### ISSUE TITLE
beehive context_channel.go sendToGroup select 

### ISSUE THREAD

**Issue by lvfei103650** [kubeedge/kubeedge#4439]
**What happened and what you expected to happen**:

What is the purpose of changing here？
I understand that it will block here，no difference
![image](https://user-images.githubusercontent.com/32949553/203775841-c84adcca-ab62-4379-b66f-eaab2f64fe76.png)

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- KubeEdge version(e.g. `cloudcore --version` and `edgecore --version`):

**Comment by wackxu** [comment]
yes  I think there is no difference, I think we can remove the  ```ch<-message``` in default case since the channel is alreay full,
